app/routes/titiler/integrated_alerts.py

```diff
@@ -0,0 +1,93 @@
+import os
+from typing import Optional, Tuple
+
+from aenum import Enum, extend_enum
+from fastapi import APIRouter, Depends, Query, Response
+from titiler.core.resources.enums import ImageType
+from titiler.core.utils import render_image
+
+from ...crud.sync_db.tile_cache_assets import get_versions
+from ...models.enumerators.tile_caches import TileCacheType
+from ...models.enumerators.titiler import IntegratedAlertConfidence, RenderType
+from .. import DATE_REGEX, raster_xyz
+from .algorithms.integrated_alerts import IntegratedAlerts
+from .readers import AlertsReader
+
+DATA_LAKE_BUCKET = os.environ.get("DATA_LAKE_BUCKET")
+
+router = APIRouter()
+
+dataset = "gfw_integrated_alerts"
+
+
+class GfwIntegratdAlertsVersions(str, Enum):
+    """GFW Integrated Alerts versions.
+
+    When using `latest` call will be redirected (307) to version tagged
+    as latest.
+    """
+
+    latest = "latest"
+
+
+_versions = get_versions(dataset, TileCacheType.cog)
+for _version in _versions:
+    extend_enum(GfwIntegratdAlertsVersions, _version, _version)
```

Comment:
Can you add comments on what GfwIntegratedAlertsVersions() is for?  I don't see it used anywhere, so then it seems like a lot of work to create an enum with all the integrated alerts versions going back for several years.

Reply:
I have added back the use of that enum in the view function which will show to the user the available valid versions to use.